# 
# Submitted-By: Robert Schwebel, 2006-02-21
# Committed-By: Robert Schwebel
#
# Error (while building xorg-server): 
#
# /opt/crosstool-0.38/gcc-3.4.4-glibc-2.3.5/i586-unknown-linux-gnu/lib/gcc/i586-unknown-linux-gnu/3.4.4/../../../../i586-unknown-linux-gnu/bin/ld: warning: libfreetype.so.6, needed by /home/rsc/svn/ptxdist-0.10-trunk/projects/visbox/local/i586-unknown-linux-gnu/usr/lib/libXfont.so, not found (try using -rpath or -rpath-link)
#
# Description: 
#
# Requires.local doesn't seem to work. 
#
# State:
# 
# unknown
#

diff -urN libXfont-X11R7.0-1.0.0-orig/xfont.pc.in libXfont-X11R7.0-1.0.0/xfont.pc.in
--- libXfont-X11R7.0-1.0.0-orig/xfont.pc.in	2005-11-19 08:15:41.000000000 +0100
+++ libXfont-X11R7.0-1.0.0/xfont.pc.in	2006-02-21 20:10:54.000000000 +0100
@@ -6,8 +6,6 @@
 Name: Xfont
 Description: X font Library
 Version: @VERSION@
-Requires: xproto fontsproto
-Requires.private: fontenc @FREETYPE_REQUIRES@
+Requires: xproto fontsproto fontenc @FREETYPE_REQUIRES@
 Cflags: -I${includedir}
-Libs: -L${libdir} -lXfont
-Libs.private: -lz -lm
+Libs: -L${libdir} -lXfont -lz -lm
